From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 21 Apr 2016 01:37:11 +0200
Subject: kmix_showeverywhere

===================================================================
---
 kmix.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kmix.desktop b/kmix.desktop
index 3e1df09..3069100 100755
--- a/kmix.desktop
+++ b/kmix.desktop
@@ -1,7 +1,6 @@
 [Desktop Entry]
 Exec=kmix -caption %c %i
 X-DocPath=kmix/index.html
-OnlyShowIn=KDE;
 Type=Application
 Terminal=false
 Icon=kmix
