From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 21 Apr 2016 01:37:11 +0200
Subject: kubuntu_cmake-policy

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 458eb9c..3aee4d3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,6 +28,8 @@ if(POLICY CMP0046)
   cmake_policy (SET CMP0046 NEW)
 endif()
 
+cmake_policy (SET CMP0046 OLD)
+
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11") 
 
 if (KMIX_KF5_BUILD)
