commit 69703e92424ef29f33070201169bcfd12f8c17cd
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Fri Mar 14 16:04:34 2014 +0000

    add interpreter to script

diff --git a/apps/kmixremote b/apps/kmixremote
index 51f4eec..2f21209 100755
--- a/apps/kmixremote
+++ b/apps/kmixremote
@@ -1,3 +1,4 @@
+#!/bin/sh
 #################################################################################
 # kmixremote - control kmix from a script.
 #
